### DISCUSSION TITLE
IKE multiple tunnels

### DISCUSSION THREAD

**Discussion by liranodiz** [strongswan/strongswan#492] [General]
I am using strongswan version 5.6.2 as initiator VS cisco device as responder.
what is the reason for multiple tunnels for each connection? how can this be avoided? 

ipsec statusall:
[root@localhost noam]# ipsec statusall
Status of IKE charon daemon (strongSwan 5.6.2, Linux 4.19.157-1.el7.x86_64, x86_64):
  uptime: 4 minutes, since Jul 01 14:10:44 2021
  malloc: sbrk 3006464, mmap 0, used 623280, free 2383184
  worker threads: 11 of 16 idle, 5/0/0/0 working, job queue: 0/0/0/0, scheduled: 36
  loaded plugins: charon aes des rc2 sha2 sha1 md5 random nonce x509 revocation constraints pubkey pkcs1 pkcs7 pkcs8 pkcs12 pgp dnskey sshkey pem openssl fips-prf curve25519 xcbc cmac hmac attr kernel-netlink resolve socket-default stroke vici updown xauth-generic counters
Listening IP addresses:
  192.168.124.1
  172.17.232.152
  172.16.3.219
  172.17.3.180
  101.101.101.1
  100.100.60.1
  169.251.0.17
  169.252.0.11
  169.253.0.211
  169.254.0.192
Connections:
VR0_tunnel32_2:  172.16.3.219...172.16.0.32  IKEv2, dpddelay=60s
VR0_tunnel32_2:   local:  [172.16.3.219] uses pre-shared key authentication
VR0_tunnel32_2:   remote: [172.16.0.32] uses pre-shared key authentication
VR0_tunnel32_2:   child:  0.0.0.0/0 === 0.0.0.0/0 TUNNEL, dpdaction=clear
VR0_tunnel31_1:  172.16.3.219...172.16.0.31  IKEv2, dpddelay=60s
VR0_tunnel31_1:   local:  [172.16.3.219] uses pre-shared key authentication
VR0_tunnel31_1:   remote: [172.16.0.31] uses pre-shared key authentication
VR0_tunnel31_1:   child:  0.0.0.0/0 === 0.0.0.0/0 TUNNEL, dpdaction=clear
VR0_tunnel33_3:  172.17.3.180...172.16.0.33  IKEv2, dpddelay=60s
VR0_tunnel33_3:   local:  [172.17.3.180] uses pre-shared key authentication
VR0_tunnel33_3:   remote: [172.16.0.33] uses pre-shared key authentication
VR0_tunnel33_3:   child:  0.0.0.0/0 === 0.0.0.0/0 TUNNEL, dpdaction=clear
VR0_tunnel34_4:  172.17.3.180...172.16.0.34  IKEv2, dpddelay=60s
VR0_tunnel34_4:   local:  [172.17.3.180] uses pre-shared key authentication
VR0_tunnel34_4:   remote: [172.16.0.34] uses pre-shared key authentication
VR0_tunnel34_4:   child:  0.0.0.0/0 === 0.0.0.0/0 TUNNEL, dpdaction=clear
Routed Connections:
VR0_tunnel34_4{99}:  ROUTED, TUNNEL, reqid 35
VR0_tunnel34_4{99}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel33_3{98}:  ROUTED, TUNNEL, reqid 31
VR0_tunnel33_3{98}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel31_1{97}:  ROUTED, TUNNEL, reqid 36
VR0_tunnel31_1{97}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel32_2{96}:  ROUTED, TUNNEL, reqid 25
VR0_tunnel32_2{96}:   0.0.0.0/0 === 0.0.0.0/0
Security Associations (8 up, 0 connecting):
VR0_tunnel31_1[11]: ESTABLISHED 13 seconds ago, 172.16.3.219[172.16.3.219]...172.16.0.31[172.16.0.31]
VR0_tunnel31_1[11]: IKEv2 SPIs: 58ed33aec782d2db_i* 3cfe56a0c7e188c0_r, pre-shared key reauthentication in 17 minutes
VR0_tunnel31_1[11]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel31_1{107}:  INSTALLED, TUNNEL, reqid 36, ESP SPIs: cf6497ff_i d43999ef_o
VR0_tunnel31_1{107}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 15 minutes
VR0_tunnel31_1{107}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel34_4[10]: ESTABLISHED 38 seconds ago, 172.17.3.180[172.17.3.180]...172.16.0.34[172.16.0.34]
VR0_tunnel34_4[10]: IKEv2 SPIs: be86204dffea95d9_i* 2426c70887c802c0_r, pre-shared key reauthentication in 22 minutes
VR0_tunnel34_4[10]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel34_4[10]: Tasks queued: CHILD_CREATE CHILD_CREATE CHILD_CREATE
VR0_tunnel34_4{92}:  INSTALLED, TUNNEL, reqid 35, ESP SPIs: c86b7031_i 01f25328_o
VR0_tunnel34_4{92}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 13 minutes
VR0_tunnel34_4{92}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel33_3[7]: ESTABLISHED 36 seconds ago, 172.17.3.180[172.17.3.180]...172.16.0.33[172.16.0.33]
VR0_tunnel33_3[7]: IKEv2 SPIs: 20701584cdf10f94_i* 7c5561c8f1910341_r, pre-shared key reauthentication in 24 minutes
VR0_tunnel33_3[7]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel33_3{95}:  INSTALLED, TUNNEL, reqid 31, ESP SPIs: cece0099_i 72f53a71_o
VR0_tunnel33_3{95}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 15 minutes
VR0_tunnel33_3{95}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel33_3[8]: ESTABLISHED 41 seconds ago, 172.17.3.180[172.17.3.180]...172.16.0.33[172.16.0.33]
VR0_tunnel33_3[8]: IKEv2 SPIs: 6820abd98ea4613c_i* 25888cf5b61b0c6a_r, pre-shared key reauthentication in 24 minutes
VR0_tunnel33_3[8]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel33_3{82}:  INSTALLED, TUNNEL, reqid 31, ESP SPIs: cf8c8bbd_i 2f803e79_o
VR0_tunnel33_3{82}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 11 minutes
VR0_tunnel33_3{82}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel33_3[9]: ESTABLISHED 40 seconds ago, 172.17.3.180[172.17.3.180]...172.16.0.33[172.16.0.33]
VR0_tunnel33_3[9]: IKEv2 SPIs: c07b39b6a810d37f_i* ea81e93b180193b8_r, pre-shared key reauthentication in 25 minutes
VR0_tunnel33_3[9]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel33_3{83}:  INSTALLED, TUNNEL, reqid 31, ESP SPIs: cacacfc1_i 907d03c0_o
VR0_tunnel33_3{83}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 11 minutes
VR0_tunnel33_3{83}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel32_2[4]: ESTABLISHED 44 seconds ago, 172.16.3.219[172.16.3.219]...172.16.0.32[172.16.0.32]
VR0_tunnel32_2[4]: IKEv2 SPIs: 699334c5ad3490bc_i* 9695914f17eeda63_r, pre-shared key reauthentication in 19 minutes
VR0_tunnel32_2[4]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel32_2{67}:  INSTALLED, TUNNEL, reqid 25, ESP SPIs: c22dbc02_i 0a90ce7a_o
VR0_tunnel32_2{67}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 16 minutes
VR0_tunnel32_2{67}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel32_2[5]: ESTABLISHED 43 seconds ago, 172.16.3.219[172.16.3.219]...172.16.0.32[172.16.0.32]
VR0_tunnel32_2[5]: IKEv2 SPIs: 89651c7917668650_i* 08c4fddaa562b123_r, pre-shared key reauthentication in 18 minutes
VR0_tunnel32_2[5]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel32_2{68}:  INSTALLED, TUNNEL, reqid 25, ESP SPIs: c00c491c_i e490b8d1_o
VR0_tunnel32_2{68}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 12 minutes
VR0_tunnel32_2{68}:   0.0.0.0/0 === 0.0.0.0/0
VR0_tunnel32_2[6]: ESTABLISHED 43 seconds ago, 172.16.3.219[172.16.3.219]...172.16.0.32[172.16.0.32]
VR0_tunnel32_2[6]: IKEv2 SPIs: 1185e8415901108e_i* 0d26d5e429910f25_r, pre-shared key reauthentication in 23 minutes
VR0_tunnel32_2[6]: IKE proposal: AES_CBC_256/HMAC_SHA2_512_256/PRF_HMAC_SHA2_512/ECP_384
VR0_tunnel32_2{69}:  INSTALLED, TUNNEL, reqid 25, ESP SPIs: c365a2f6_i 9ee11dac_o
VR0_tunnel32_2{69}:  AES_CBC_256/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 11 minutes
VR0_tunnel32_2{69}:   0.0.0.0/0 === 0.0.0.0/0

/etc/ipsec.conf:
`[root@localhost noam]# cat /etc/ipsec.conf
conn %default
        keyexchange = ikev1
        authby = secret
        type = tunnel
        ike = aes256-sha2_512-ecp384!
        dpdaction = clear
        dpddelay = 60
        dpdtimeout = 180
        mobike = no

conn VR0_tunnel32_2
        keyexchange = ikev2
        left = 172.16.3.219
        leftsubnet = 0.0.0.0/0
        right = 172.16.0.32
        rightsubnet = 0.0.0.0/0
        esp = aes256-sha2_256-ecp384!
        lifetime = 1800
        ikelifetime = 2100
        aggressive = no
        type = tunnel
        lifebytes = 4608000000
        auto = route
        leftsourceip = %config4
        mark = 2
        leftupdown = /home/dnfv/utils/dpd_hook/run.py

conn VR0_tunnel31_1
        keyexchange = ikev2
        left = 172.16.3.219
        leftsubnet = 0.0.0.0/0
        right = 172.16.0.31
        rightsubnet = 0.0.0.0/0
        esp = aes256-sha2_256-ecp384!
        lifetime = 1800
        ikelifetime = 2100
        aggressive = no
        type = tunnel
        lifebytes = 4608000000
        auto = route
        leftsourceip = %config4
        mark = 1
        leftupdown = /home/dnfv/utils/dpd_hook/run.py

conn VR0_tunnel33_3
        keyexchange = ikev2
        left = 172.17.3.180
        leftsubnet = 0.0.0.0/0
        right = 172.16.0.33
        rightsubnet = 0.0.0.0/0
        esp = aes256-sha2_256-ecp384!
        lifetime = 1800
        ikelifetime = 2100
        aggressive = no
        type = tunnel
        lifebytes = 4608000000
        auto = route
        leftsourceip = %config4
        mark = 3
        leftupdown = /home/dnfv/utils/dpd_hook/run.py

conn VR0_tunnel34_4
        keyexchange = ikev2
        left = 172.17.3.180
        leftsubnet = 0.0.0.0/0
        right = 172.16.0.34
        rightsubnet = 0.0.0.0/0
        esp = aes256-sha2_256-ecp384!
        lifetime = 1800
        ikelifetime = 2100
        aggressive = no
        type = tunnel
        lifebytes = 4608000000
        auto = route
        leftsourceip = %config4
        mark = 4
        leftupdown = /home/dnfv/utils/dpd_hook/run.py`

**Comment by Thermi** [comment]
Provide logs like in the topic linked in the big red banner before opening a discussion, or #196 (same topic)
